src/indexes.rs

```diff
@@ -1,7 +1,7 @@
 use crate::{
     client::Client,
     documents::{DocumentQuery, DocumentsQuery, DocumentsResults},
-    errors::Error,
+    errors::*,
```

Comment:
```suggestion
    errors::{Error, MeilisearchCommunicationError},
```
I'm not sure it's the case in Rust but it's not better to just import what you need?

Reply:
It's a design choice indeed, since there are already a lot of `*` in these imports I left it like that. But in the future we might want to change all `*` to specific imports.

Reply:
changed nonetheless!